### ISSUE TITLE
Addition of more semantic features

### ISSUE THREAD

**Issue by clarkenj** [lingualab/lingualabpy#9]
1. Semantic diversity (semD)
- A measure of the distance between various contexts in which a word appears. Low diversity values indicate that a word is used in very narrow contexts (e.g. spinach), while high values indicate usage in more diverse contexts (e.g. predicament, and function words).  [(Hoffman et al., 2014)](https://www.sciencedirect.com/science/article/abs/pii/S001094521200322X?via%3Dihub)
- Implementation based on latent semantic analysis reported in [Hoffman et al. (2013)](https://link.springer.com/article/10.3758/s13428-012-0278-x). SemD values for 31,741 English words are provided.
- Correlates with other psycholinguistic measures (frequency, imageability).
- Also reflects ambiguity - lower value words are less ambiguous.

TO BE CONT.

**Comment by SophiePellerin** [comment]
I agree that 1 and 2 essentially get to the same thing, maybe keeping just 1 would be enough. 3 is fine to keep because it gets to something different (semantic similarity regardless of whether the words are used in the same contexts or not), it's very possible or even likely that semantically similar words are used in similar contexts but it's also likely not always the case.
